lib/isochrone-tile.js

```diff
@@ -42,11 +42,11 @@ export default function isochroneTile (imageData, {height, scaleFactor, surface,
         val = surface[ypsurf * width + xpsurf]
       }
 
-      let colorScheme = colorScheme || function (val) {
+      const _colorScheme = colorScheme || function (val) {
```

Comment:
I'm not a fan of prefixing variables with underscores. Can we just say `colorScheme = colorScheme || ...`?